pyaerocom/web/helpers_evaluation_iface.py

```diff
@@ -467,6 +529,279 @@ def _init_ts_data():
         yearly_mod = []
         )
 
+def _create_diurnal_weekly_data_object(coldata,resolution):
```

Comment:
This is a very useful method and should probably migrated elsewhere at some point, perhaps directly as a class method of `ColocatedData`. However, for now I think we can leave it this way.

Reply:
See #219